src/notation/internal/notationparts.cpp

```diff
@@ -22,6 +22,7 @@
 #include "notationparts.h"
 
 #include "libmscore/score.h"
+#include "libmscore/keylist.h"
```

Comment:
Needed?

Reply:
Not anymore, good catch.
In the first version the code which is now in `Score::getKeyList()` was in `NotationParts::appendStaves()` and then this line was necessary. When moved to a dedicated method this line wasn't removed.